CONTRIBUTING.md

```diff
@@ -5,10 +5,10 @@ Thank you for taking the time to contribute to the Augur Project! Before you sta
 Make sure to be using Python 2.7.xx, Serpent 2.0.2, and Ethereum 1.6.1. It is recommended to clone from the master branch for both [Pyethereum](https://github.com/ethereum/pyethereum) and [Serpent](https://github.com/ethereum/serpent). Pyethereum may install Ethereum 1.4.0 -- uninstall the latter if it happens. Check your version of Ethereum by running this on terminal: `pip show ethereum | grep Version`
 Check your version of python by running this: `python --version`
  
-### Pull Requests and Issues
+###Pull Requests and Issues
 When making [pull requests](https://help.github.com/articles/creating-a-pull-request/), follow the pull request template in the root directory. When [raising issues](https://help.github.com/articles/creating-an-issue/), follow the appropriate issue template in the root directory. Make sure commit messages are informative and concise.
  
-### Communication
+###Communication
```

Comment:
I don't exactly know how that happened, but it did... is there any way to get rid of this part?